api/HTMLBaseElement.json

```diff
@@ -114,6 +114,41 @@
               "deprecated": false
             }
           }
+        },
+        "forbid_tab_newline_lessthan_urls": {
```

Comment:
```suggestion
        "forbid_special_characters": {
```

Reply:
I'm OK with this. If they add more special chars we can reconsider.